javascript/builtins/Array.json

```diff
@@ -2256,7 +2256,14 @@
                 "version_added": false
               },
               "nodejs": {
-                "version_added": false
+                "version_added": "6.5.0",
+                "notes": "--harmony does not work, --harmony-array-prototype-values is required",
```

Comment:
I think the `"flags"` block is sufficient to get this across, so this note can be dropped

Reply:
I added this note because _most_ `--harmony-*` flags are subsumed under the `--harmony` flag, so many users might expect that this is _always_ the case - but it isn't. For example, `Array.prototype.includes` was available since Node.js `5.0.0` with the `--harmony-array-includes` or the `--harmony` flags, but _some_ flags like `--harmony-array-prototype-values` are _not_ (or not yet) subsumed under `--harmony`.

I think the note is helpful, but not vital. If you want, I'll remove it.

Reply:
Ah, OK. I didn't quite get that distinction (I've since [read up on it](https://nodejs.org/en/docs/es6/)). Maybe we can rework the note to use the Node.js docs' terminology? What do you think of this? `<code>--harmony-array-prototype-values</code> is in progress and is not enabled by the <code>--harmony</code> flag.`

Reply:
Sure! Much better. Done.